iotfunctions/anomaly.py

```diff
@@ -272,7 +257,7 @@ class SpectralAnomalyScore(BaseTransformer):
     '''
     def __init__(self, input_item, windowsize, output_item):
         super().__init__()
-        logger.debug(input_item)
+        logger.info(input_item)
```

Comment:
It is better to place debug level log for finding out input.

Reply:
done